x/ibc/core/spec/01_concepts.md

```diff
@@ -7,6 +7,69 @@ order: 1
 > NOTE: if you are not familiar with the IBC terminology and concepts, please read
 this [document](https://github.com/cosmos/ics/blob/master/ibc/1_IBC_TERMINOLOGY.md) as prerequisite reading.
 
+## Client Creation, Updates, and Upgrades
+
+IBC clients are on chain light clients. The light client is responsible for verifying
+counterparty state. A light client can be created by any user submitting a client
+identifier and a valid initial `ClientState` and `ConsensusState`. The client identifier
+must not already be used. Clients are given a client identifier prefixed store to
+store their associated client state and consensus states. Consensus states are 
+stored using their associated height. 
+
+Clients can be updated by any user submitting a valid `Header`. The client state callback
+to `CheckHeaderAndUpdateState` is responsible for verifying the header against previously
+stored state. The function should also return the updated client state and consensus state 
+if the header is considered a valid update. A light client, such as Tendermint, may have
+client specific parameters like `TrustLevel` which must be considered valid in relation
+to the `Header`.
+
+Clients may be upgraded. The upgrade should be verified using `VerifyUpgrade`. It is not
+a requirement to allow for light client upgrades. For example, the solo machine client 
+will simply return an error on `VerifyUpgrade`.
+
+## Client Misbehaviour
+
+IBC clients must freeze when the counterparty chain becomes malicious and 
+takes actions that could fool the light client into accepting invalid state 
+transitions. Thus, relayers are able to submit Misbehaviour proofs that prove 
+that a counterparty chain has signed two Headers for the same height. This 
+constitutes misbehaviour as the IBC client could have accepted either header 
+as valid. Upon verifying the misbehaviour the IBC client must freeze at that 
+height so that any proof verifications for the frozen height or later fail.
+
+Note, there is a difference between the chain-level Misbehaviour that IBC is 
+concerned with and the validator-level Evidence that Tendermint is concerned 
+with. Tendermint must be able to detect, submit, and punish any evidence of 
```

Comment:
I'd prob include a reference to the tendermint evidence docs here

Reply:
this was existing docs (just got moved around). @AdityaSripal would know better than I on what TM evidence docs to link